openapi/generator_test.go

```diff
@@ -733,6 +734,27 @@ func TestSetServers(t *testing.T) {
 	assert.Equal(t, servers, g.API().Servers)
 }
 
+type customUnit struct {
```

Comment:
I'd like to switch this custom type to an `int` or `float64`, to demonstrate and test that scalar go types that implement the `Exampler` interface can also provide their own example value.

The type could still be a struct, if we wanted to have a more complex type, such as:
```
type customUnit struct {
   value    float64
   currenty string
}
```
but that would be overkill for just tests.